src/components/Form/Form.stories.mdx

```diff
@@ -774,6 +774,8 @@ This field implements the [**FieldDef**](#generic-field-props-fielddef) interfac
 | **`placeholder`** | `string` | optional - The hint displayed in the input before the user enters a value. |
 | **`prefixElement`** | `JSX.Element` | optional - Icon at the beginning of the text field. |
 | **`type`** | `string` | optional - Type of the input element. It should be a [valid HTML5 input type](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Form_%3Cinput%3E_types). |
+| **`maxRows`** | `string `\|` number` | optional - Maximum number of rows to display when multiline options is set to true. |
```

Comment:
I know MUI's documentation says it can be string or number, but I believe we should keep it 100% as number (for both min and max).
```suggestion
| **`maxRows`** | `string` | optional - Maximum number of rows to display when multiline options is set to true. |
```